### PR TITLE
Added user and role entities

### PR DESCRIPTION
**Pull request by apokhyla-inc**
Added user and role entities, enabled ddl-auto property for hibernate.

### CODE REVIEW

File: src/main/java/com/example/app/entity/Role.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import lombok.Data;
+
+/**
+ * Role entity.
+ */
+
+@Entity
+@Table(name = "role")
+@Data
```
Comment (Pasichniuk) [inline src/main/java/com/example/app/entity/Role.java]: Consider swapping @Data and @Entity for better readability.

File: src/main/java/com/example/app/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+/**
+ * User entity.
+ */
+
+@Entity
+@Table(name = "user")
+@Data
```
Comment (Pasichniuk) [inline src/main/java/com/example/app/entity/User.java]: Same.

**Review by Pasichniuk — COMMENTED** [review]
It is not too much, but it is honest work. Please fix mentioned points.

**Comment by Pasichniuk** [comment]
I don't know what to write here.

File: src/main/java/com/example/app/entity/Role.java (hunk 1 above)
Comment (apokhyla-inc) [reply src/main/java/com/example/app/entity/Role.java]: Agree, fixed.

File: src/main/java/com/example/app/entity/User.java (hunk 2 above)
Comment (apokhyla-inc) [reply src/main/java/com/example/app/entity/User.java]: Fixed.

**Comment by apokhyla-inc** [comment]
Thanks a lot for the review, fixed mentioned moments.

**Pull request merged**